.github/workflows/publish.yml

```diff
@@ -12,9 +12,9 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: 3.10
```

Comment:
@ocefpaf is it an issue to change the version number in the publish workflow? If we're unsure I'd suggest not changing this.

Reply:
We can use "3.x" here.